validator/sawtooth_validator/server/cli.py

```diff
@@ -194,17 +200,22 @@ def main(args=sys.argv[1:]):
                            human_readable_name='Log'):
         init_errors = True
 
+    if validator_config.public_uri is None:
```

Comment:
This also needs to handle the case of network_endpoint specifying interface names (e.g. tcp://eth0:8800).